network/ddclient-configuration.sh

```diff
@@ -56,55 +56,13 @@ then
 fi
 
 # Enter your Hostname
-while :
-do
-    HOSTNAME=$(input_box "Please enter the Host that you want to configure DDNS for.\nE.g. 'example.com'")
-    if ! yesno_box_yes "Is this correct? $HOSTNAME"
-    then
-        msg_box "OK, please try again."
-    else
-        if [ -z "$HOSTNAME" ]
-        then
-            msg_box "Please don't leave the inputbox empty."
-        else
-            break
-        fi
-    fi
-done
+HOSTNAME=$(input_box_flow "Please enter the Host that you want to configure DDNS for.\nE.g. 'example.com'")
 
 # Enter your login
-while :
-do
-    LOGIN=$(input_box "Please enter the login for your DDNS provider.\nIt will be most likely the domain or registered email address depending on your DDNS Provider.\nE.g. 'example.com' or 'mail@example.com'\nIf you are not sure, please refer to the documentation of your DDNS provider.")
-    if ! yesno_box_yes "Is this correct? $LOGIN"
-    then
-        msg_box "OK, please try again."
-    else
-        if [ -z "$LOGIN" ]
-        then
-            msg_box "Please don't leave the inputbox empty."
-        else
-            break
-        fi
-    fi
-done
+LOGIN=$(input_box_flow "Please enter the login for your DDNS provider.\nIt will be most likely the domain or registered email address depending on your DDNS Provider.\nE.g. 'example.com' or 'mail@example.com'\nIf you are not sure, please refer to the documentation of your DDNS provider.")
 
 # Enter your password
-while :
-do
-    PASSWORD=$(input_box "Please enter the password or api-key that you've got for DynDNS from your DDNS provider.\nIf you are not sure, please refer to the documentation of your DDNS provider.")
-    if ! yesno_box_yes "Is this correct? $PASSWORD"
-    then
-        msg_box "OK, please try again."
-    else
-        if [ -z "$PASSWORD" ]
```

Comment:
`input_box_flow` doesn't take this into account?

Reply:
because it is not really necessary. (we show the result box later on and there you should make yourself sure that all values are correct). If not it is the users mistake and not ours.

Reply:
I think we should handle that case anyway. 

Just added a fix.